### ISSUE TITLE
Tabs are slightly longer in 8.5.4 than in 8.5.3

### ISSUE THREAD

**Issue by ja231c** [notepad-plus-plus/notepad-plus-plus#13834]
### Description of the Issue
This is at 150% scaling in Windows, I have not checked 100% scaling.
8.5.4 has a slightly larger font than 8.5.3 causing tabs to be longer.
Longer tabs take more space on the screen, that results in less tabs being visible on the screen at the same time.
Also the gap between the newly enlarged icon and the text seems to be slightly more than previously, further elongating the tab.
The gap between the end of text and the close tab button also seems to be larger.

### Expected Behavior
Tabs are exactly the same size as in the previous version.

### Actual Behavior
Tabs are slightly longer and take more space on the screen.

**Comment by alankilborn** [comment]
> Expected behavior: Tabs are exactly the same size as in the previous version.

I don't know that you have the right to expect this.  Software changes.

**Comment by anandv85** [comment]
I Agree with [ja231c](https://github.com/ja231c).  The font used in the tab title are larger compare to previous.  This is so annoying to me.  Please revert back to the same font size as the previous version 8.5.3.

**Comment by sterpumihai** [comment]
How can a tab title font change be "annoying"? I know OCD is a disease but come on!

**Comment by anandv85** [comment]
Any update on this? I still see the font size for the tab is larger.  Would be nice if this can be fixed.

**Comment by anandv85** [comment]
> How can a tab title font change be "annoying"? I know OCD is a disease but come on!

OCD is a disorder, not disease :-)

The tab size are growing when we open multiple files with larger file names because of the font size.  That was the main intension and nothing to do with OCD :-)

**Comment by pedroos** [comment]
I use a lot of tabs and the change subtracted about one inch from my available text buffer. I change documents using NavigateTo and don't need an increase in tab size or tab text size. The change made a significant negative impact on usability in my daily scenario and I suggest a feature for tab size and/or tab text size configuration.

**Comment by alankilborn** [comment]
@pedroos If you use _NavigateTo_ plugin, suggest you turn off tabs entirely?  Then their size won't bother you, and you get more screen real estate for editing!
